src/mixins/text/_text.scss

```diff
@@ -0,0 +1,46 @@
+@import "~@moda/tokens";
+@import "font-smoothing";
+
+$treatments: body h1 h2 h3 h4;
+
+@mixin text($treatment: body) {
```

Comment:
Would be nice if the offset values for both serif and sans are the same. If not you'll have to alter them depending on the type treatment. @SamJacobs

Reply:
👍